syntax = "proto3";

import "google/protobuf/timestamp.proto";

message UserDocument {
    // Object id from MongoDB
    string userid = 1;
    
    // Name of this user
    string username = 2;

    // Password of this user
    string password = 3;

    // Creation timestamp
    google.protobuf.Timestamp created_at = 4;
  }
